src/main/java/com/artipie/rpm/CliArguments.java

```diff
@@ -46,27 +46,57 @@
     /**
      * Cli options.
      */
-    private final Options options;
+    private static final Options OPTIONS = new Options()
+        .addOption(RpmOptions.DIGEST.option())
+        .addOption(RpmOptions.NAMING_POLICY.option())
+        .addOption(RpmOptions.FILELISTS.option());
+
+    /**
+     * Cli.
+     */
+    private final CommandLine cli;
 
     /**
      * Ctor.
+     * @param cli Command line
      */
-    public CliArguments() {
-        this (
-            new Options()
-                .addOption(RpmOptions.DIGEST.option())
-                .addOption(RpmOptions.NAMING_POLICY.option())
-                .addOption(RpmOptions.FILELISTS.option())
-        );
+    public CliArguments(final CommandLine cli) {
+        this.cli = cli;
     }
 
     /**
      * Ctor.
+     * @param args Command line arguments
+     */
+    public CliArguments(final String... args) {
+        this(CliArguments.parsed(args));
+    }
+
+    /**
+     * Repository.
      *
-     * @param options Cli options.
+     * @return Repository.
+     * @throws IllegalArgumentException If the arg value is incorrect
+     */
+    public Path repository() {
+        final List<String> args = this.cli.getArgList();
+        if (args.size() != 1) {
```

Comment:
@olenagerasimova I think this should be `args.size() == 0` because we can have more than one argument declared.